app/main/handle-external-link.ts

```diff
@@ -125,16 +131,30 @@ export default function handleExternalLink(
       url: url.href,
       downloadPath,
       async completed(filePath: string, fileName: string) {
+        const notificationTitle = t.__("Download Complete");
+        const notificationBody = t.__(
+          "Click to view the folder where the file ({{{fileName}}}) was downloaded.",
+          {
+            fileName,
+          },
+        );
         const downloadNotification = new Notification({
-          title: t.__("Download Complete"),
-          body: t.__("Click to show {{{fileName}}} in folder", {fileName}),
+          title: notificationTitle,
+          body: notificationBody,
           silent: true, // We'll play our own sound - ding.ogg
         });
         downloadNotification.on("click", () => {
           // Reveal file in download folder
           shell.showItemInFolder(filePath);
         });
         downloadNotification.show();
+        latestDownloadPath = filePath;
+        send(
+          contents,
+          "show-download-success",
+          notificationTitle,
+          notificationBody,
+        );
```

Comment:
This seems to do both the old notification mechanism and the new one -- is that intendedd? 

It is important for backwards-compatibility to make sure we do the old thing on existing web app versions. But I don't know if doing both would be problematic, and in any case, I would expect some sort of detection logic and comments explaining the reasoning behind what we're doing.

Reply:
I think we don't want to send both. There are some potential advantages to the native notification; It appears even if your desktop app is not focused -- say if you started downloading a long video and then tabbed away.

Is there a way to check if the native notification actually sent?

Reply:
> Is there a way to check if the native notification actually sent?

I checked this and there's not for all OS. Only windows emits a failed event: https://www.electronjs.org/docs/latest/api/notification#event-failed-windows.

Might be ok having both notifications in that case?